core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowVersionResource.scala

```diff
@@ -343,4 +347,55 @@ class WorkflowVersionResource {
       res
     }
   }
+
+  @POST
+  @Produces(Array(MediaType.APPLICATION_JSON))
+  @Path("/clone/{vid}")
+  @RolesAllowed(Array("REGULAR", "ADMIN"))
+  def cloneVersion(
+      @PathParam("vid") vid: UInteger,
+      @Auth sessionUser: SessionUser,
+      requestBody: java.util.Map[String, Int]
+  ): UInteger = {
+    val offset = requestBody.get("offset")
```

Comment:
Rename all of the variable name `offset` to `displayedVersionId`. I don't like the name `offset`.